src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,56 +1,129 @@
-    private Joystick armJoystick = new Joystick(Constants.JOYSTICK_PORT2);
-
-    private final Drivetrain drivetrain = new Drivetrain();
-    private final SimpleArm simpleArm = new SimpleArm();
-
-    private final PowerDistribution powerDistribution = new PowerDistribution();
-    private final AutoUtils autoUtils = new AutoUtils();
-    
-    public RobotContainer() {
-        powerDistribution.clearStickyFaults();
-        startDefaultCommands();
-    }
-
-    public void startDefaultCommands() {
-        drivetrain.setDefaultCommand(new SimpleDriveCommand(drivetrain, 
-        () -> -joystick.getRawAxis(1),
-        () -> -joystick.getRawAxis(0)));
-
-        simpleArm.setDefaultCommand(new RunCommand(() -> simpleArm.moveArm(getArmToggle())));
-    }
-
-    public int getArmToggle() {
-        if (armJoystick.getRawButton(6)) {
-            return 6;
-        } else if (armJoystick.getRawButton(7)) {
-            return 7;
-        } else {
-            return -1;
-        }
-    }
-
-    public Drivetrain getRobotDrive() {
-        return drivetrain;
-    }
-
-    public AutoUtils getAutoPath() {
-        return autoUtils;
-    }
-
-    public Command getAutonomousCommand() {
-        return new RunCommand(() -> drivetrain.testDrive(0.1, 0.1), drivetrain).withTimeout(5);
-    }
+  // The robot's subsystems
+  private final DriveSubsystem m_robotDrive;
+
+  // The driver's controller
+  XboxController m_driverController = new XboxController(OIConstants.kDriverControllerPort);
+
+  /**
+   * The container for the robot. Contains subsystems, OI devices, and commands.
+   */
+  public RobotContainer() {
+    // Configure the button bindings
+    //configureButtonBindings();
+
+    m_robotDrive = new DriveSubsystem();
+
+    // Configure default commands
+    m_robotDrive.setDefaultCommand(
+        // The left stick controls translation of the robot.
+        // Turning is controlled by the X axis of the right stick.
+        new RunCommand(
+            () -> m_robotDrive.drive(
+                MathUtil.applyDeadband(-m_driverController.getLeftY(), 0.06),
+                MathUtil.applyDeadband(-m_driverController.getLeftX(), 0.06),
+                MathUtil.applyDeadband(-m_driverController.getRightX(), 0.06),
+                MathUtil.applyDeadband(-m_driverController.getRightY(), 0.06),
+                m_driverController.getRightBumper(), m_driverController.getAButton()),
+            m_robotDrive));
+  }
+
+  /**
+   * Use this method to define your button->command mappings. Buttons can be
+   * created by
+   * instantiating a {@link edu.wpi.first.wpilibj.GenericHID} or one of its
+   * subclasses ({@link
+   * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then calling
+   * passing it to a
+   * {@link JoystickButton}.
+   */
+  private void configureButtonBindings() {
+    new JoystickButton(m_driverController, Button.kR1.value)
+        .whileTrue(new RunCommand(
```

Comment:
You could change the condition to onTrue so the driver only needs to press the button once to stall the chassis.